src/aiogram_dialog/widgets/kbd/state.py

```diff
@@ -117,6 +121,7 @@ def __init__(
             result: Any = None,
             on_click: Optional[OnClick] = None,
             when: WhenCondition = None,
+            show_mode: ShowMode = ShowMode.AUTO,
```

Comment:
let's do it in the same way as in manager methods:
```py
show_mode: Optional[ShowMode] = None,
```

Reply:
done